Gemfile

```diff
@@ -29,7 +29,10 @@ gem "activejob-uniqueness", require: "active_job/uniqueness/sidekiq_patch"
 gem "fog-aws"
 gem "sys-filesystem"
 
+gem "decidim-faceless", git: "https://github.com/digidemlab/decidim-module-faceless", branch: "release/0.26-stable"
 gem "letter_opener_web", "~> 1.3"
+gem "omniauth-oauth2", ">= 1.4.0", "< 2.0"
```

Comment:
Does the gem `omniauth-oauth2` and `sprockets` are required for this feature ? 

If it works as expected without, I think we can remove it

Reply:
After reviewing, gems are deleted !

Thanks for the review !